pylint/plugins/hass_logger.py

```diff
@@ -58,21 +58,22 @@ def visit_call(self, node):
 
         first_arg = node.args[0]
 
-        if not isinstance(first_arg, astroid.Const):
+        if not isinstance(first_arg, astroid.Const) or not first_arg.value:
             return
 
         log_message = first_arg.value
 
-        if not len(log_message):
+        if not len(log_message) < 2:
             return
 
-        if log_message[-1] == ".":
+        if log_message[-1] == "." and log_message[-2] != ".":
+            # Trailing ... is permitted
```

Comment:
Not sure we should allow this. We don't write full sentences so we don't need to use ellipsis. It doesn't add value in my opinion.

Reply:
I'll change the comment to to say its not permitted but currently not checked for as there are too many to clean up on the first pass.

Reply:
I'm probably overthinking this. I think I can just strip out the `...` and they all still make sense.

Reply:
I was overthinking it. They still make sense with it stripped out. I've done that for most of them and will cleanup anything else pylint catches.